tests/test_listener.py

```diff
@@ -0,0 +1,72 @@
+# Source pytest module
+# Copyright (C) 2020  Ngô Xuân Minh
+#
+# This file is part of palace.
+#
+# palace is free software: you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published
+# by the Free Software Foundation, either version 3 of the License,
+# or (at your option) any later version.
+#
+# palace is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with palace.  If not, see <https://www.gnu.org/licenses/>.
+
+"""This pytest module tries to test the correctness of the class Listener."""
+
+from palace import Listener
+from pytest import raises
+
+from math import inf
+
+
+def test_gain(context):
+    """Test write property gain."""
+    Listener(context).gain = 5/7
+    Listener(context).gain = 7/5
+    Listener(context).gain = 0
+    Listener(context).gain = inf
+    with raises(ValueError): Listener(context).gain = -1
+
+
+def test_position(context):
+    """Test write property position."""
+    Listener(context).position = (1, 0, 1)
+    Listener(context).position = (1, 0, -1)
+    Listener(context).position = (1, -1, 0)
+    Listener(context).position = (1, 1, 0)
+    Listener(context).position = (0, 0, 0)
+    Listener(context).position = (1, 1, 1)
```

Comment:
`context.listener.position = 1, 1, 1` would look a lot cleaner; the same with velocity.